.github/workflows/test.yml

```diff
@@ -1,43 +1,43 @@
 name: Test
 
-on: [push, pull_request]
+# https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662
```

Comment:
Would this config stop tests on non-main branches?

Reply:
This would stop tests on push for non-main branches, only run when they put a PR on main.

This is also an important callout - if we change our core branch name again this is another piece to maintain. thoughts?

Reply:
have we changed the core branch name in the past? Is there a reason to think we might do so again?

Reply:
Forgot to respond here!

It changed once from master -> main, but I feel like it'd be pretty stable after that? Just the idea that the head branch **can** change